src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -108,8 +108,10 @@ The APIs used to retrieve the values are:
 * [GCMemoryInfo.TotalCommittedBytes](https://docs.microsoft.com/dotnet/api/system.gcmemoryinfo.totalcommittedbytes):
   Gets the total committed bytes of the managed heap.
 
-* [GC.GetGCMemoryInfo().GenerationInfo[i].SizeAfterBytes](https://docs.microsoft.com/dotnet/api/system.gcgenerationinfo):
-  Represents the size in bytes of a generation on exit of the GC reported in GCMemoryInfo.
+* Heap size is retrieved with an internal method `GC.GetGenerationSize`,
```

Comment:
I would add to this comment that we are doing this explicitly to workaround a known issue in .NET 6. Here is the link to the PR that fixed it: https://github.com/dotnet/runtime/pull/60309